apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget.cs

```diff
@@ -127,6 +127,52 @@ protected override void InitializeTarget()
             base.InitializeTarget();
         }
 
+        private Func<object, Value> BuildProtoConverter()
+        {
+            // Create reusable JsonSerializer to reduce allocations
+            var jsonSettings = new Newtonsoft.Json.JsonSerializerSettings() { ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore };
+            jsonSettings.Converters.Add(new Newtonsoft.Json.Converters.StringEnumConverter());
+            var jsonSerializer = Newtonsoft.Json.JsonSerializer.CreateDefault(jsonSettings);
+            return new Func<object, Value>(o => {

```

Comment:
Please use the simpler syntax with the open-brace on its own line:
``` C#
return o =>
{
    ...
};
```

Reply:
Fixed